controllers/controllers.news.js

```diff
@@ -72,3 +79,15 @@ exports.postComments = (req, res, next) => {
       next(err);
     });
 };
+
+exports.removeComment = (req, res, next) => {
+  const { comment_id } = req.params;
+  deleteComment(comment_id)
+    .then(() => {
+      res.status(204).send();
```

Comment:
if you are not sending a body (which you actually can't with a `204` status!) then you can instead use `res.sendStatus(204)`